Add torrijas recipe (#519)
LukeSmithxyz/based.cooking: src/torrijas.md

# Torrijas (A spanish eastern treat)

> Traditional Spanish desert, consumed especially during the catholic season called "Lent". As a curiosity, they used to be given as a gift to parturient women. You will have delicious sweet toasts.  There are many torrijas recipes, but this one is a keeper.

Reviewer: ```suggestion
Traditional Spanish dessert, consumed especially during the catholic season called "Lent". As a curiosity, they used to be given as a gift to parturient women. You will have delicious sweet toasts.  There are many torrijas recipes, but this one is a keeper.
```

> # Torrijas (A spanish eastern treat)
> 
> Traditional Spanish dessert, consumed especially during the catholic season called "Lent". As a curiosity, they used to be given as a gift to parturient women. You will have delicious sweet toasts.  There are many torrijas recipes, but this one is a keeper.

Reviewer: ```suggestion
# Torrijas (A spanish eastern treat)

![torrijas](pix/torijas.webp)

Traditional Spanish dessert, consumed especially during the catholic season called "Lent". As a curiosity, they used to be given as a gift to parturient women. You will have delicious sweet toasts.  There are many torrijas recipes, but this one is a keeper.
```

Image added in. You have to accept this though. I don't have the privileges to change your PRs for you.

	-  Cooking oil
	-  1 strip of lemon peel
	-  1 strip orange peel
	-  2 teaspoons of cinnamon
-  Optional: 
	-  3 tablespoons of honey
	-  1 teaspoon of cardamom seeds
	-  1 star anise

## Directions

1. Bring the milk, ½ cup of sugar, lemon and orange peel, and spices (cardamom seeds and star anise) to a slow simmer.
1. Cut the bread in thick slices.
1. When the milk mixture has been simmering for about **20/25 minutes**, turn off the heat and soak the slices of bread in this mixture. Be careful not to completely wet them to the point that they will break apart, but try to get them to absorb as much milk as possible.
1. Let the slices of wet bread rest and cool.
1. Beat the eggs in a shallow bowl and dip the bread slices in the egg mixture. In the meantime, heat up about ½ an inch of the olive oil in a deep, heavy pan on medium-high heat.
1. Fry the slices two by two, flipping them halfway so that both sides are nice and crisp.
1. Let the torrijas rest on paper towels to absorb excess oil. In another bowl mix the remaining sugar (1/2 cup) with the cinnamon.
1. Cover the slices in the cinnamon sugar mixture.
1. Add the honey (you can add more or less depending on preference).

> 1. Refrigerate the torrijas the time you want, or just enjoy them freshly made, **(I recommend to taste them both ways)**.

Reviewer: ```suggestion
1. Refrigerate the torrijas for as long as you want, or just enjoy them freshly made, **(I recommend to taste them both ways)**.
```

Reviewer: Can you make this change as well?

    
## Contribution

> - ![XMR Logo](https://lukesmith.xyz/pix/xmr.svg) `86BjG6aLhRALso4ovSQNBhHTsuZHy1qW8XRdztDm1VMn8NDxS5SbGVn4Hyow9aRiTsaMttuZCzRoq2BKQ2Wy5VVH8SVgrRg`

Reviewer: You can put your name here if you want.

Author: Ey mate I forgot to add the photo of the dish already prepared.
Could you incorporate it above ingredients?
I am not very familiar with github.
The photo is in the commit at: `data/pix/torrijas.webp`
Best regards.
